proxy/certs/certs.go

```diff
@@ -169,8 +169,9 @@ func (s *RemoteCertSource) Local(instance string) (ret tls.Certificate, err erro
 		return ret, err
 	}
 
-	p, _, n := util.SplitName(instance)
-	req := s.serv.SslCerts.CreateEphemeral(p, n,
+	p, region, n := util.SplitName(instance)
```

Comment:
nit: probably `r` to be consistent (the variable isn't going far)

Reply:
fixed.